dev/maven-plugin/src/main/scala/com/lightbend/lagom/maven/PortRangeBean.scala

```diff
@@ -0,0 +1,8 @@
+package com.lightbend.lagom.maven
+
+import scala.beans.BeanProperty
+
+class PortRangeBean {
```

Comment:
Aha, maybe I see. The bean property setters are used to change the default when an instance of `PortRangeBean` is created. If it's guaranteed that `min/max` are written and read by the same thread, then we are good and you can disregard my above comment. Though, I wonder what would be the interest of having the same thread writing and reading values from this instance, which is why I'm being a bit suspicious here.